README.md

```diff
@@ -51,4 +53,6 @@
   make down
 ```
 
-**WARNING** This application is **still under development** and still not suitable for production purposes.
+## Release workflow

```

Comment:
The doc is fantastic BUT IMHO it is better to add this in a separate document under ``docs``. The idea is using later a tool like mkdocs or similar, bundle all pages and deploy in https://github.com/ITISFoundation/itisfoundation.github.io

The main README should be minimal and to the point! At most you can put a link to that if you like

Reply:
ok moved to ops/README.md